Update fbbi.yml

dosumis opened the pull request:
New repo

jamesaoverton commented:
I was getting an HTTP 400 error for the links you provided. I added `/master/`to them and I think that I fixed them. Please confirm, and then I'll merge.

dosumis commented:
> I was getting an HTTP 400 error for the links you provided. I added /master/to them and I think that I fixed them. Please confirm, and then I'll merge.

Ooops. Yes. That's correct.  Thanks!

Robbie1977 commented:
@jamesaoverton As I think David (@dosumis) is tied up in meetings I'll confirm the fix looks good now.
